src/components/common/StyledModal.tsx

```diff
@@ -0,0 +1,172 @@
+      border-bottom: 0.1rem solid var(--color-background-banner);
+    }
+    
+    .modali-header .modali-title {
+      font-size: inherit;
+      font-weight: inherit;
+      line-height: 1;
+    }
+    
+    .modali-header .modali-close-button {
+      background: transparent;
+      font-size: 4rem;
+      line-height: 1;
+      color: var(--color-modali-close);
+      font-weight: normal;
+      opacity: .5;
+      padding: 0;
+      margin: auto 0 auto auto;
+      
+        &:hover {
+          opacity: 1;
+        }
+    }
+    
+    .modali-body {
+      font-size: 1.4rem;
+      line-height: 1.3;
+      background: inherit;
+      color: var(--color-text-primary);
+    }
+    
+    .modali-footer {
+      margin: 1.6rem auto 0;
+      height: 5.6rem;
+      border-top: 0.1rem solid var(--color-background-banner);
+      width: 100%;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      padding: 0 1.6rem;
+      box-sizing: border-box;
+
+      .modali-button.modali-button-default, .modali-button.modali-button-cancel {
+        margin: 0;
+        border-radius: 0.6rem;
+        outline: 0;
+        height: 3.6rem;
+        box-sizing: border-box;
+        letter-spacing: 0.03rem;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        text-decoration: none;
+        text-transform: uppercase;
+        background: var(--color-background-CTA);
+        color: var(--color-text-CTA);
+        font-weight: var(--font-weight-bold);
+        font-size: 1.4rem;
+        
+        &:hover {
+          background-color: var(--color-background-button-hover);
+          color: var(--color-text-button-hover);
+        }
+      }
+      
+      .modali-button.modali-button-default {
+        border-radius: 0.6rem;
+        min-width: 14rem;
+        padding: 0 1.6rem;
+      }
+
+      .modali-button.modali-button-cancel {
+        background: transparent;
+        color: var(--color-text-active);
+      }
+    }
+  }
+  
+  // Heavier specification of the selector to ensure override from global.ts
+  // Ideally take out the modali default injected styles and only load from our end.
+  body.modali-open .modali.modali-size-large {
+    width: 80vw;
+    max-width: initial;
+    min-width: initial;
+    max-height: 100%;
+      @media (min-width: 500px) {
+          min-width: initial !important;
+      }
+      @media ${MEDIA.mobile},
+      @media ${MEDIA.tablet} {
+        width: 100%;
+      }
+  }
+  
+  body.modali-open .modali-wrapper-centered {
+    top: 0!important;
+  }
+  
+  body.modali-open .modali-body-style {
+    padding: 0;
+  }
+`
+
+const Modal: React.FC<Modali.ModalProps> = (props) => (
+  <>
+    <ModaliGlobalStyle />
+    <Modali.Modal {...props} />
+  </>
+)
+
+// To import default Modali and not change much code elsewhere
+const StyledModali = { ...Modali, Modal }
+
+export { StyledModali as default, ModalHook, toggleModaliComponent, useModali }
```

Comment:
I would declare the export of the modaly stuff on the top of the file, right after the exports.
I we could just do:
`export { ModalHook, toggleModaliComponent as toggleModalComponent, useModali as useModal } from 'modali'`

Reply:
Can't.
You need to import to get variables into the module scope. This is an error:
```js
export {useModali} from 'modali'
console.log(useModali)
```
 and this is just duplicating code:
```js
export { ModalHook, toggleModaliComponent as toggleModalComponent, useModali as useModal } from 'modali'
import { ModalHook, toggleModaliComponent as toggleModalComponent, useModali as useModal } from 'modali'
```

Reply:
I'm all for renaming vars to _useModal~i~_ at `export` or `import` time

Reply:
From what I understand my code here is correct.. right?

Reply:
Right.
Unless... @anxolin ?

Reply:
Well, I don't think mixing import and export is duplicating code.

In my opinion is more clear what we are doing, reexported some things renamed, and exporting a restyled Modal object.

```js
import { Modal } from 'modali'
export { ModalHook, toggleModaliComponent as toggleModalComponent, useModali as useModal } from 'modali'
```